### DISCUSSION TITLE
GraphGym - Only final-epoch test results are logged, not best val epochs

### DISCUSSION THREAD

**Discussion by KerenRozen** [pyg-team/pytorch_geometric#9951] [Q&A]
Hi,
I'm using GraphGym for a graph-level classification task, and I'm trying to report test metrics at the epoch where the validation accuracy is highest. However, after training finishes, I only see test results from the final epoch in my logs. 
I noticed that in `pytorch_geometric/graphgym/results/example_node_grid_example/agg/`, there is an example of both final-epoch and best-val-epoch test metrics being reported, indicating this should be possible.
Is there a specific config setting  that I need to enable so that test metrics are also evaluated/logged at best val epoch?
Thanks for your help!
